Pointer error on creating kafka topic

```shell
 rhoas kafka topics create --name jsdc
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1455e39]
```

--- a/pkg/cmd/kafka/topics/create/create.go
+++ b/pkg/cmd/kafka/topics/create/create.go
@@ -33,8 +33,9 @@ type Options struct {
 // NewCreateTopicCommand gets a new command for creating kafka topic.
 func NewCreateTopicCommand(f *factory.Factory) *cobra.Command {
 	opts := &Options{
-		Config: f.Config,
-		Logger: f.Logger,
+		Connection: f.Connection,
+		Config:     f.Config,
+		Logger:     f.Logger,
 	}
 
 	cmd := &cobra.Command{

--- a/pkg/sdk/kafka/topics/topics.go
+++ b/pkg/sdk/kafka/topics/topics.go
@@ -69,7 +69,7 @@ func brokerConnect(opts *Options) (broker *kafka.Conn, ctl *kafka.Conn, err erro
 
 	sdkkafka.TransformResponse(&kafkaInstance)
 
-	conn, err := dialer.Dial("tcp", *kafkaInstance.BootstrapServerHost)
+	conn, err := dialer.Dial("tcp", kafkaInstance.GetBootstrapServerHost())
 	if err != nil {
 		return nil, nil, err
 	}

